examples/meta/generator/parse.py

```diff
@@ -201,6 +192,25 @@ def p_argumentList(self, p):
 
         p[0] = {"ArgumentList": arguments}
 
+    def p_argumentList_nonEmpty(self, p):
```

Comment:
Not sure about this capitalisation underscore mix

Reply:
haha yeah you're right. Will make this consistent

Reply:
Initial underscores `t_` and `p_` are enforced by the parsing library, that's why there is some confusion in this file